README.md

```diff
@@ -140,6 +140,34 @@ module.exports = {
 
 These JSDoc custom tags are used to adjust generated pact interactions.
 
+<a name="pact-axios"></a>
```

Comment:
So it could be referenced in error thrown here https://github.com/HLTech/pact-gen-ts/pull/66/files#diff-bc6cfaa7a5c070b320c0254626ccd4d2e6c8a83d3c07ea40554dcfb4e4b43536R45

Reply:
What if we define each option in readme as a low-level header, h5 or h6. Github then automatically creates visible anchors for them

Reply:
I thought that headers cannot be styled with `ticks` - but luckly they can! I will make tags as headers, that would be definitely a nice improvement :)